packages/core/src/components/StructuredMetadataTable/StructuredMetadataTable.tsx

```diff
@@ -25,22 +25,22 @@ import {
   MetadataListItem,
 } from './MetadataTable';
 
-const listStyle = {
+const listStyle = createStyles({
   root: {
     margin: '0 0',
     listStyleType: 'none',
   },
-};
+});
 
-const nestedListStyle = {
+const nestedListStyle = createStyles({
   root: {
     ...listStyle.root,
     paddingLeft: '8px',
```

Comment:
This is why there is a weird indentation. I don't know how we would like to handle the nested lists :thinking: